### DISCUSSION TITLE
The extension does not display properly in the Yandex browser for Android

### DISCUSSION THREAD

**Discussion by bauw2008** [uBlockOrigin/uBlock-issues#3123] [General]
I'm using the latest version of Yandex Browser for Android, but this browser seems to be officially restricting the use of the official version of uBlock Origin, so I'm using the uBlock Origin development build, but the latest version of the extension is not displayed properly.
![Screenshot_20240215-145530](https://github.com/uBlockOrigin/uBlock-issues/assets/77882800/5fd92f3d-4162-481c-bd8a-281e829041a4)

**Comment by gorhill** [comment]
Do you have the same issue if you use [uBO Lite](https://github.com/uBlockOrigin/uBOL-home?tab=readme-ov-file#ubo-lite)?

**Reply by bauw2008** [reply]
Using uBO Lite extensions is a normal extension display.
uBlock Origin development build version is incomplete, I tried to modify some of the content in the css, and it can barely be used.

**Reply by gorhill** [reply]
From your screenshot, I am guessing the issue is that uBO assumes its popup panel fills the viewport when it detects it runs on mobile, but the Yandex browser is not giving it a whole-screen viewport but rather the usual browser-sized popup panel as when rendering on desktop. In mobile mode, there is no minimum width set since uBO is meant to fill the viewport with a `100%` width. `100%` doesn't work though for a floating popup panel.

Maybe a CSS rule such as follow would fix the issue:

```css
html:mobile {
    min-width: 360px;
}
```

Is this something you can try on your side?

**Reply by bauw2008** [reply]
Your guess should be correct, other Android browsers like Firefox and Kiwi have a semi-fullscreen setting on the pop-up panel, while Yandex is centered on the pop-up panel displaying status, which is more beautiful to use than semi-full-screen. Thank you for your kind reply.

**Reply by mrxyzl** [reply]
Hi, can you clarify on how to modified the css for the change?
